readme.md

```diff
@@ -9,15 +9,19 @@ Self-sovereign identity (SSI) is a term used to describe the digital movement th
 
 ## Contents
 
-- [🤔 Philosophy & Concepts](#-philosophy--concepts)
-- [📄 Technical Specifications](#-technical-specifications)
-- [🛠 Tools & Technologies](#-tools--technologies)
-- [💁‍♂️ Products & Services](#️-products--services)
-- [📡 Networks](#-networks)
-- [📚 Resources](#-resources)
-- [📆 Events](#-events)
-- [💪 Contribute](#-contribute)
-- [🖋️ License](#️-license)
+
+- [Awesome Self-Sovereign Identity](#awesome-self-sovereign-identity)
+  - [Contents](#contents)
```

Comment:
Is this extra TOC nesting intentional?

Reply:
No, probably happened on save

Reply:
Assuming you're using the Markdown All in One extension for VSCode, you can fix this by adding ` "markdown.extension.toc.levels": "2..6"` to your settings :)